### PR TITLE
Invoice suppression fix

### PR DESCRIPTION
**Pull request by calellowitz**
@nickpell @benrudolph 
This turns back on disable on submit for accounting forms, and should fix all the ones that broke.
http://manage.dimagi.com/default.asp?190406#1067397
http://manage.dimagi.com/default.asp?190458#1067806

### CODE REVIEW

File: corehq/apps/accounting/forms.py [hunk 1]
```diff
@@ -1977,7 +2009,8 @@ def resend_email(self):
 
 
 class SuppressInvoiceForm(forms.Form):
-    submit_kwarg = 'suppress_invoice'
+    submit_kwarg = 'suppress'
+    suppress = forms.CharField(widget=forms.HiddenInput, required=False)
```
Comment (benrudolph) [inline corehq/apps/accounting/forms.py]: :joy:

**Comment by benrudolph** [comment]
lg2m when tests pass

**Comment by nickpell** [comment]
:+1: thanks Cal!

**Comment by calellowitz** [comment]
:white_check_mark:

**Pull request merged**